Dockerfile

```
@@ -0,0 +1,15 @@
+FROM python:3.8 AS xaitk_base
+
+# install poetry and configure to install to pip environment
+RUN pip install poetry
+RUN poetry config virtualenvs.create false
```

These can be collapsed into the same build layer (good practice, reduces result image size)
```suggestion
RUN pip install poetry \
 && poetry config virtualenvs.create false
```